react/index.js

```diff
@@ -27,36 +28,36 @@ export { default as Section } from './Section/Section';
 
 // Icons
 export { default as ChevronIcon } from './ChevronIcon/ChevronIcon';
-export { default as DownRightArrowIcon } from './DownRightArrowIcon/DownRightArrowIcon';
 export { default as ClearIcon } from './ClearIcon/ClearIcon';
 export { default as CloseIcon } from './CloseIcon/CloseIcon';
+export { default as CrossIcon } from './CrossIcon/CrossIcon';
+export { default as DeleteIcon } from './DeleteIcon/DeleteIcon';
+export { default as DownRightArrowIcon } from './DownRightArrowIcon/DownRightArrowIcon';
+export { default as ErrorIcon } from './ErrorIcon/ErrorIcon';
+export { default as FacebookIcon } from './FacebookIcon/FacebookIcon';
 export { default as HeartIcon } from './HeartIcon/HeartIcon';
+export { default as HelpIcon } from './HelpIcon/HelpIcon';
+export { default as LinkedInIcon } from './LinkedInIcon/LinkedInIcon';
+export { default as MailIcon } from './MailIcon/MailIcon';
+export { default as PlusIcon } from './PlusIcon/PlusIcon';
 export { default as ProfileIcon } from './ProfileIcon/ProfileIcon';
 export { default as SearchIcon } from './SearchIcon/SearchIcon';
 export { default as StarIcon } from './StarIcon/StarIcon';
 export { default as ThumbsUpIcon } from './ThumbsUpIcon/ThumbsUpIcon';
-export { default as ErrorIcon } from './ErrorIcon/ErrorIcon';
-export { default as TickIcon } from './TickIcon/TickIcon';
 export { default as TickCircleIcon } from './TickCircleIcon/TickCircleIcon';
-export { default as HelpIcon } from './HelpIcon/HelpIcon';
-export { default as PlusIcon } from './PlusIcon/PlusIcon';
-export { default as DeleteIcon } from './DeleteIcon/DeleteIcon';
-export { default as MailIcon } from './MailIcon/MailIcon';
-export { default as CrossIcon } from './CrossIcon/CrossIcon';
-export { default as FacebookIcon } from './FacebookIcon/FacebookIcon';
-export { default as LinkedInIcon } from './LinkedInIcon/LinkedInIcon';
+export { default as TickIcon } from './TickIcon/TickIcon';
 export { default as TwitterIcon } from './TwitterIcon/TwitterIcon';
 
 // Fields
-export { default as TextField } from './TextField/TextField';
-export { default as EmailField } from './EmailField/EmailField';
-export { default as ClearField } from './TextField/ClearField/ClearField';
 export { default as Autosuggest } from './Autosuggest/Autosuggest';
-export { default as Textarea } from './Textarea/Textarea';
 export { default as Checkbox } from './Checkbox/Checkbox';
-export { default as MonthPicker } from './MonthPicker/MonthPicker';
+export { default as ClearField } from './TextField/ClearField/ClearField';
 export { default as Dropdown } from './Dropdown/Dropdown';
+export { default as EmailField } from './EmailField/EmailField';
+export { default as MonthPicker } from './MonthPicker/MonthPicker';
 export { default as SlideToggle } from './SlideToggle/SlideToggle';
+export { default as TextField } from './TextField/TextField';
+export { default as Textarea } from './Textarea/Textarea';
```

Comment:
Hmm, that's interesting I used unix's `sort` program to do this and it looks like it makes a distinction between a capital letter.

I.e. a capital is greater then a non capital. 

(Learn something new everyday)